duplicate symbol _xine_builtin_plugin_info in:
    .libs/load_plugins.o
    .libs/builtins.o
https://bugs.xine-project.org/show_bug.cgi?id=579
--- src/xine-engine/builtins.h.orig	2017-02-21 05:13:13.000000000 -0600
+++ src/xine-engine/builtins.h	2017-03-04 06:22:57.000000000 -0600
@@ -31,7 +31,7 @@
 
 #include <xine/xine_internal.h>
 
-const plugin_info_t xine_builtin_plugin_info[7];
+extern const plugin_info_t xine_builtin_plugin_info[7];
 
 #endif
 
